fighthealthinsurance/asgi.py

```
@@ -42,56 +42,70 @@
+    if tags.get("fuzz_guard") is True:
+        return None
+
+    # Check transaction name for fuzz guard middleware
+    transaction = event.get("transaction", "")
+    if "FuzzGuardMiddleware" in transaction:
+        return None
+
+    # Check for specific gRPC error messages from Ray
+    message = event.get("message", "") or ""
+    if "Logstream proxy failed to connect" in message:
+        logger.warning(f"Ray logstream proxy connection failed (filtered from Sentry)")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)